src/main/java/org/jabref/gui/EntryTypeView.java

```diff
@@ -168,4 +182,117 @@ private void setEntryTypeForReturnAndClose(Optional<BibEntryType> entryType) {
         viewModel.stopFetching();
         this.close();
     }
+
+    //The description is coming from biblatex manual chapter 2
+    //Biblatex documentation is favored over the bibtex,
+    //since bibtex is a subset of biblatex and biblatex is better documented.
+    public String getDescription(BibEntryType selectedType) {
```

Comment:
You could e.g. declare this method static, then you can access it in the other class as well. (Or you could create some helper class with this static method, or probably some different location for it.)

Reply:
@systemoperator I have implemented your first suggestion! Please let me know what you think!